src/components/landingpage/TwoColumn.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import { Box, Flex, Image } from "@chakra-ui/core"
+
+const TwoColumn = ({ image, alt, imagePosition="left", children, ...props }) => {
+  const boxProps = {
+    p: 50,
+    pr: 0,
+    width: [ '100%', '100%', '100%', '50%' ],
+  }
+
+  if (imagePosition === "left") {
```

Comment:
if my memory serves me right, I think you can actually tell the Flex component and do 

`flexDirection={ imagePostion === 'left' ? 'row-reverse' : 'row' }`

instead of having to build the layout 2 times